MAUI/DataForm/migration.md

```diff
@@ -888,6 +906,86 @@ The following properties of the `Xamarin SfDataForm` is moved to the [DataFormDe
+</tr>
+
+<tr>
+<td>{{'[FormatString](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormNumericItemBase.html#Syncfusion_XForms_DataForm_DataFormNumericItemBase_FormatString)'| markdownify }}</td>
+<td>CustomFormat</td>
+<td>Gets or sets the format used to specify the formatting of editor value.</td>
+</tr>
+
+<tr>
+<td>{{'[Maximum](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormNumericUpDownItem.html#Syncfusion_XForms_DataForm_DataFormNumericUpDownItem_Maximum)'| markdownify }}</td>
+<td>Maximum</td>
+<td>Gets or sets the numerical maximum value for DataFormNumericEditor.</td>
+</tr>
+
+<tr>
+<td>{{'[Minimum](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormNumericUpDownItem.html#Syncfusion_XForms_DataForm_DataFormNumericUpDownItem_Minimum)'| markdownify }}</td>
+<td>Minimum</td>
+<td>Gets or sets the numerical minimum value for DataFormNumericEditor.</td>
+</tr>
+
+<tr>
+<td>{{'[CultureInfo](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormNumericItemBase.html#Syncfusion_XForms_DataForm_DataFormNumericItemBase_CultureInfo)'| markdownify }}</td>
+<td>Culture</td>
+<td>Gets or sets the culture for DataFormNumericEditor.</td>
+</tr>
+
+</table>
+
+#### DataFormMaskedTextItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[CultureInfo](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormMaskedEditTextItem.html#Syncfusion_XForms_DataForm_DataFormMaskedEditTextItem_CultureInfo)'| markdownify }}</td>
+<td>Culture</td>
+<td>Gets or sets the culture for mask values based on the given culture.</td>
+</tr>
+
+<tr>
+<td>{{'[Mask](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormMaskedEditTextItem.html#Syncfusion_XForms_DataForm_DataFormMaskedEditTextItem_Mask)'| markdownify }}</td>
+<td>Mask</td>
+<td>Gets or sets the mask value which restricts the inputs provided in the control.</td>
+</tr>
+
+<tr>
+<td>{{'[MaskType](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormMaskedEditTextItem.html#Syncfusion_XForms_DataForm_DataFormMaskedEditTextItem_MaskType)'| markdownify }}</td>
+<td></td>
+<td>Gets or sets the mask type for the DataFormMaskedTextEditor.</td>
+</tr>
+
+<tr>
+<td>{{'[PromptChar](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormMaskedEditTextItem.html#Syncfusion_XForms_DataForm_DataFormMaskedEditTextItem_PromptChar)'| markdownify }}</td>
+<td>PromptChar</td>
+<td>Gets or sets a char which used as a prompt char of DataFormMaskedTextEditor.</td>
+</tr>
+
+<tr>
+<td>{{'[ValueMaskFormat](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormMaskedEditTextItem.html#Syncfusion_XForms_DataForm_DataFormMaskedEditTextItem_ValueMaskFormat)'| markdownify }}</td>
+<td>ValueMaskFormat</td>
+<td>Gets or sets the format for the value.</td>
```

Comment:
<td>Get or set the format for the value.</td>

Reply:
Content changed